### PR TITLE
Database server seeding

### PR DESCRIPTION
**Pull request by LittleJennie**
Set up basic server, created database connection, created seeding file for database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # reservations
 Reservation component on the Open Table restaurant page. 
+
+
+### Initial Database Seeding
+
+1. Run the following script in terminal with your mysql password with in this project's root directory. This script will create the `reservation` database in your mySQL server.
```
Comment (primanbo) [inline README.md]:
```suggestion
1. Run the following script in terminal with your mysql password within this project's root directory. This script will create the `reservation` database in your mySQL server. 
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # reservations
 Reservation component on the Open Table restaurant page. 
+
+
+### Initial Database Seeding
+
+1. Run the following script in terminal with your mysql password with in this project's root directory. This script will create the `reservation` database in your mySQL server. 
+    `mysql < ./server/db/Schema.sql -u root -p`
+
+2. Exist mysql terminal, and run the following script after turning on the server in a separate terminal. This script will create `reservations_table` and will also generate 100 random restaurant data in this table.
```
Comment (primanbo) [inline README.md]:
```suggestion
2. Exit MySQL terminal, and run the following script after turning on the server in a separate terminal. This script will create `reservations_table` and will also generate 100 random restaurant data in this table.
```

File: server/db/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('reservations', 'root', 'password', {
+  host: 'localhost',
+  dialect: 'mysql'
+});
+
+const Reservations_table = sequelize.define('reservations_table', {
+    // attributes
+    restaurant_id: {
```
Comment (primanbo) [inline server/db/index.js:10]: Indentations: lines 4-5 are indented with 2 spaces, but line 9 and onward use a 4 space indentation style... try to be consistent. This is an easy fix :)

I think the HR style guide recommends 2 spaces.

File: server/db/seed.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+const Reservations = require('./index.js');
+const _restaurant_name_id = require('./restaurantData.js');
+const moment = require('moment');
+
+// only available for the next 30 days reservation
+const createSeed = (days, data = []) => {
+    let dateCount = days;
```
Comment (primanbo) [inline server/db/seed.js:7]: @LittleJennie If you want to follow the [HR style guide](https://learn.makerpass.com/groups/hrsf119/courses/reactorcore/docs.student-wiki?id=style-guide#indentation), 
 change indentations from 4 spaces to 2.

File: server/db/seed.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    return data;
+};
+
+const insertToReservationTable = () => {
+    const seed = createSeed(90);
+
+    Reservations.Reservations_table.bulkCreate(seed)
+    .then(() => console.log('data created'))
```
Comment (primanbo) [inline server/db/seed.js:55]: @LittleJennie, indent when using multi-line chaining for better readability? Found a few good examples here: https://github.com/airbnb/javascript#whitespace--chains

File: server/db/restaurantData.js [hunk 6]
```diff
@@ -0,0 +1,404 @@
+const _restaurant_name_id = [   
+    {
```
Comment (primanbo) [inline server/db/restaurantData.js:2]: [check indentation style guide ](https://github.com/airbnb/javascript#whitespace--spaces)

**Review by primanbo — COMMENTED** [review]
@LittleJennie Looks good overall! Just check the indentation style for code consistency.

File: server/db/index.js (hunk 3 above)
Comment (LittleJennie) [reply server/db/index.js:10]: indentations are fixed in the next PR

File: server/db/seed.js (hunk 4 above)
Comment (LittleJennie) [reply server/db/seed.js:7]: indentations are fixed in the next PR

File: server/db/seed.js (hunk 5 above)
Comment (LittleJennie) [reply server/db/seed.js:55]: indentations are fixed in the next PR

File: server/db/restaurantData.js (hunk 6 above)
Comment (LittleJennie) [reply server/db/restaurantData.js:2]: indentations are fixed in the next PR

**Pull request merged**